src/com/mambu/apisdk/util/URLHelper.java

```diff
@@ -24,11 +24,12 @@
 @Singleton
 public class URLHelper {
 
+	public static final String AMPERSAND_DELIMITER = "&";

```

Comment:
can be private

Reply:
That was the intent, but I missed it :)
Thanks